contribution-guidelines.md

```diff
@@ -32,6 +32,8 @@ Please take a moment to check whether an issue already exists. If it does, it ne
 
 We are always thrilled to receive pull requests, and do our best to process them as quickly as possible. Not sure if that typo is worth a pull request? Do it! We will appreciate it.
 
+If you don't get some response to a pull request within a day, please do mention it in the #ipfs channel, in case it has gotten forgotten.
```

Comment:
"If you don't get a response to a pull request within a few days, feel free to mention it in the #ipfs IRC channel. We may be able to get to it faster." Sound better? 

Doesn't put the pressure of a day on us. It also probably wasn't forgotten; we've got a lot of priorities.